### PR TITLE
Add SNI and ServerIP to tester input

### PR DESCRIPTION
**Pull request by jacobhaven**
This is needed to test lazy-loading with the `gokeyless-tester`

### CODE REVIEW

File: client/keys.go [hunk 1]
```diff
@@ -13,11 +13,9 @@ import (
...
 // PrivateKey represents a keyless-backed RSA private key.
 type PrivateKey struct {
-	public crypto.PublicKey
-	ski    gokeyless.SKI
-	digest gokeyless.Digest
-
-	client *Client
+	public   crypto.PublicKey
+	client   *Client
+	template *gokeyless.Operation
```
Comment (lziest) [inline client/keys.go]: do you think "opTemplate" would be better? Naming is hard, and  English is my 2nd language. :cry:

Comment (jacobhaven) [inline client/keys.go]: Maybe, It's just so long

Comment (kisom) [inline client/keys.go]: You also see `tpl` a lot. I'd be fine with that.

Comment (jacobhaven) [inline client/keys.go]: I'm really just fine with `template`. Any other comments?

Comment (lziest) [inline client/keys.go]: do you like to create a new Metadata struct here? and call it md? Decouple from gokeyless.Operation? key.template just confuses me a little bit.

Comment (jacobhaven) [inline client/keys.go]: That's fair. It was mainly in order to be extensible with other/new `gokeyless.Operation` info.

**Comment by lziest** [comment]
LGTM

**Pull request merged**